### ISSUE TITLE
MARGIN represents a multiplier, not a percentage?

### ISSUE THREAD

**Issue by metaperl** [verata-veritatis/bybit-market-maker#5]
In [the settings](https://github.com/verata-veritatis/bybit-market-maker/blob/master/settings.py#L5) we see: 

    MARGIN = 10 # in percent

But this is not a percentage. 10 percent = .1

In looking at [the initial calculations](https://github.com/verata-veritatis/bybit-market-maker/blob/master/market_maker/market_maker.py#L133) we see:

    balance = self.req.get_wallet_balance() * last_price * MARGIN

so what is happening is that we are multiplying our capital by the margin, but converting it to dollars, correct?

By the way, I'm re-implementing your code using a generic crypto library:
https://github.com/metaperl/piranha-market-maker

**Comment by metaperl** [comment]
Continuing with:

    balance = self.req.get_wallet_balance() * last_price * MARGIN

Presuming a BTC balance of 0.01 and last price of 10,000, we have $100 without margin and with a `MARGIN` of 10, we have $1000 to play with... so MARGIN most definitely is mis-documented and represents a capital multiplier, correct?

**Comment by verata-veritatis** [comment]
Yeah like I've said, this entire repo is getting a big update in the coming days. A lot of it is quite messy.

**Comment by verata-veritatis** [comment]
@metaperl Hey there actually took a dive into what you said, and it is actually a percentage. On cross, your maximum margin is 100x your balance. Therefore, 10 * wallet balance would 10% of your total available margin.
